connecting to Carbon at 127.0.0.1:2003 (localhost)
onConnect: code = -1 err = 0 errstr = Connection refused errno = Connection refused
disconnection with status -1
onConnect: code = -1 err = 0 errstr = Connection refused errno = Connection refused
onConnect: code = -1 err = 1 errstr = getsockopt(SO_ERROR): Socket operation on non-socket errno = Bad file descriptor
disconnection with status -1
onConnect: code = -1 err = 1 errstr = getsockopt(SO_ERROR): Socket operation on non-socket errno = Bad file descriptor

--------------------------[Exception thrown]---------------------------
time:   2014-04-22-071201
type:   ML::Exception
pid:    4894; tid: 4894
what:   couldn't connect to Redis: Connection refused
stack:
00: 0x0x7f65799b9db7 at __cxa_throw + 0x27 in /home/rtbkit/rtbkit/build/x86_64/bin/../bin/libexception_hook.so + 0xdb7/home/rtbkit/rtbkit/build/x86_64/bin/../bin/libexception_hook.so(__cxa_throw+0x27) [0x7f65799b9db7]
01: 0x0x7f6577cb1076 at Redis::AsyncConnection::test() + 0x196 in /home/rtbkit/rtbkit/build/x86_64/bin/../bin/libredis.so + 0x1a076/home/rtbkit/rtbkit/build/x86_64/bin/../bin/libredis.so(_ZN5Redis15AsyncConnection4testEv+0x196) [0x7f6577cb1076]
02: 0x0x40f065 at main + 0x695 in build/x86_64/bin/banker_service_runner + 0xf065build/x86_64/bin/banker_service_runner(main+0x695) [0x40f065]
03: 0x0x7f657849076d at __libc_start_main + 0xed in /lib/x86_64-linux-gnu/libc.so.6 + 0x2176d/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xed) [0x7f657849076d]
04: 0x0x40f76d in build/x86_64/bin/banker_service_runner + 0xf76dbuild/x86_64/bin/banker_service_runner() [0x40f76d]

terminate called after throwing an instance of 'ML::Exception'
  what():  couldn't connect to Redis: Connection refused
